tutorials/plot_brainstorm_auditory.py

```diff
@@ -0,0 +1,350 @@
+# precomputed. To run everything from scratch change this to False.
+use_precomputed = True
+
+###############################################################################
+# The data was collected with a CTF 275 system at 2400 Hz and low-pass
+# filtered at 600 Hz. Here the data and empty room data files are read to
+# construct instances of :class:`mne.io.Raw`.
+data_path = bst_auditory.data_path()
+
+subject = 'bst_auditory'
+subjects_dir = op.join(data_path, 'subjects')
+
+raw_fname1 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_01.ds')
+raw_fname2 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_02.ds')
+erm_fname = op.join(data_path, 'MEG', 'bst_auditory',
+                    'S01_Noise_20131218_01.ds')
+preload = not use_precomputed
+raw = read_raw_ctf(raw_fname1, preload=preload)
+mne.io.concatenate_raws([raw, read_raw_ctf(raw_fname2, preload=preload)])
+raw_erm = read_raw_ctf(erm_fname, preload=preload)
+
+###############################################################################
+# Data channel array consisted of 274 MEG axial gradiometers, 26 MEG reference
+# sensors and 2 EEG electrodes (Cz and Pz).
+# In addition:
+#   - 1 stim channel for marking presentation times for the stimuli
+#   - 1 audio channel for the sent signal
+#   - 1 response channel for recording the button presses
+#   - 1 ECG bipolar
+#   - 2 EOG bipolar (vertical and horizontal)
+#   - 12 head tracking channels
+#   - 20 unused channels
+# The head tracking channels and the unused channels are marked as misc
+# channels. Here we define the EOG and ECG channels.
+raw.set_channel_types({'HEOG': 'eog', 'VEOG': 'eog', 'ECG': 'ecg'})
+if not use_precomputed:
+    # Leave out the two EEG channels for easier computation of forward.
+    raw.pick_types(meg=True, eeg=False, stim=True, misc=True, eog=True,
+                   ecg=True)
+
+###############################################################################
+# For noise reduction, a set of bad segments have been identified and stored
+# in csv files. The bad segments are later used to reject epochs that overlap
+# with them.
+# The file for the second run also contains some saccades. The saccades are
+# removed by using SSP. You can view the files with your favorite text editor.
+
+onsets = list()
+durations = list()
+descriptions = list()
+saccades = list()
+offset = raw._raw_lengths[0]
+for idx in [1, 2]:
+    with open(op.join(data_path, 'MEG', 'bst_auditory',
+                      'events_bad_0%s.csv' % idx), 'r') as f:
+        reader = csv.reader(f)
+        for row in reader:
+            onsets.append(int(row[0]) + offset * (idx - 1))
+            durations.append(int(row[1]))
+            descriptions.append(row[3])
+            if row[3] == 'saccade':  # Events for removal of saccades.
+                sample = int(row[0]) + offset * (idx - 1)
+                saccades.append([sample, 0, 1])
```

Comment:
this works but I don't consider it a good practice. We should advertise pandas for this. Eg.

```
In [25]: import pandas as pd
In [26]: df = pd.read_csv('events_bad_01.csv', header=None, names=['onset', 'duration', 'wtf', 'label'])
Out[26]:
     onset  duration  wtf label
0     7625      2776    1   BAD
1   142459       892    1   BAD
2   216954       460    1   BAD
3   345135      5816    1   BAD
4   357687      1053    1   BAD
5   409101      3736    1   BAD
6   461110       179    1   BAD
7   479866       426    1   BAD
8   764914     11500    1   BAD
9   798174      6589    1   BAD
10  846880      5383    1   BAD
11  858863      5136    1   BAD
```

then you can do df['onset'] etc...